impress-org/givewp
# Milestone now supports a featured image

_**Note: This PR is depended upon work in #5224 - it cannot be merged until that PR is first approved and merged!**_

Resolves #5227 

## Description
This PR introduces functionality for the Milestone block to support a featured image. Through the addition of an ImageControl block, the Milestone block's inspector area now includes a basic, functional featured image selection UI. This UI will need design improvements in the future, but for now it is effective at allowing a user to select an image from their Media Library or upload an image, and persisting the selected image's URL with the block.

## Affects
This PR affects the Inspector area of the Milestone block.

## Visuals
![FeaturedImage](https://user-images.githubusercontent.com/5186078/92041922-25bfa900-ed47-11ea-872a-7b3fdb7fc950.gif)

## Pre-review Checklist

## Testing Instructions
1. Add a new post or page
2. Add a Milestone block
3. Change the featured image
4. Save the page, check that the image is also reflected on the frontend
